components/DeleteModal.tsx

```diff
@@ -47,32 +42,56 @@ export function DeleteModal({
       <p className="row-start-2 row-end-2 ps-5 pt-5 text-lg w-full h-full font-bold">
         {dataToDeleteInfo}
       </p>
-      <AcceptButtonSVG
-        alt="acceptButton"
-        width={64}
-        height={64}
-        className="relative mt-3 row-start-3 row-end-3 col-start-1 col-end-1 left-5 sm:left-5"
-        onClick={() => handleDeletion()}
-        onMouseOver={(e: React.MouseEvent<SVGElement, MouseEvent>) =>
-          e.currentTarget.classList.add('[&_:nth-child(1)]:fill-yellow-400')
-        }
-        onMouseOut={(e: React.MouseEvent<SVGElement, MouseEvent>) =>
-          e.currentTarget.classList.remove('[&_:nth-child(1)]:fill-yellow-400')
-        }
-      />
-      <DeclineButtonSVG
-        alt="declineButton"
-        width={64}
-        height={64}
-        className="relative mt-3 row-start-3 row-end-3 col-start-1 col-end-1 left-32 sm:left-28"
-        onClick={() => closeModalUseState(false)}
-        onMouseOver={(e: React.MouseEvent<SVGElement, MouseEvent>) =>
-          e.currentTarget.classList.add('[&_:nth-child(1)]:fill-yellow-400')
-        }
-        onMouseOut={(e: React.MouseEvent<SVGElement, MouseEvent>) =>
-          e.currentTarget.classList.remove('[&_:nth-child(1)]:fill-yellow-400')
-        }
-      />
+      <button
+        className="border border-yellow-500 mt-3 row-start-3 row-end-3 col-start-1 col-end-1 w-[64px] h-[64px] bg-gray-300"
+        onMouseOver={(e) => {
+          e.currentTarget.classList.remove('bg-gray-300');
+          e.currentTarget.classList.add('bg-gray-600');
+        }}
+        onMouseOut={(e) => {
+          e.currentTarget.classList.remove('bg-gray-600');
+          e.currentTarget.classList.add('bg-gray-300');
+        }}
```

Comment:
Mouse over tyylittelyt on parempi tehdä `hover:bg-gray-600` classNamella suoraan buttoniin (hover on CSS ominaisuus, ei sinänsä liity Tailwindiin). Ks. [Tailwindin dokumentaatio](https://tailwindcss.com/docs/hover-focus-and-other-states#hover-focus-and-active) lisäinfoksia.